### ISSUE TITLE
TF2 Nightly-20190710 broken import

### ISSUE THREAD

**Issue by seanpmorgan** [tensorflow/tensorflow#30560]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04/19.10
- TensorFlow installed from (source or binary): Binary - **tf2-nightly-2.0-preview**
- TensorFlow version (use command below): 2.0 preview
- Python version: python3.6 (probably all others)
**Describe the current behavior**

**Describe the expected behavior**
Able to import TensorFlow

**Code to reproduce the issue**
```
pip install tf-nightly-2.0-preview
python -c "import tensorflow as tf"
```

Results in:
```
>>> import tensorflow as tf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sean-morgan/miniconda3/envs/tf-broken/lib/python3.6/site-packages/tensorflow/__init__.py", line 98, in <module>
    from tensorflow_core import *
  File "/home/sean-morgan/miniconda3/envs/tf-broken/lib/python3.6/site-packages/tensorflow_core/__init__.py", line 363, in <module>
    _API_MODULE = sys.modules[__name__].bitwise  # pylint: disable=undefined-variable
NameError: name 'sys' is not defined
```

**Comment by yongtang** [comment]
@seanpmorgan I added a PR #30562 for the fix.

**Comment by seanpmorgan** [comment]
Fixed by 7c7c449f175db71cbcdf544bb4104f84f8cb91d9 closing as this should be fixed in tomorrow's nightly.

**Comment by seanpmorgan** [comment]
Small correction. The live nightly update is working and the fix is in today's nightly 💯
